corehq/apps/reports_core/templates/reports_core/base_template_new.html

```diff
@@ -126,5 +126,11 @@ <h4>{{ report.title }}</h4>
       {% endblock reporttable %}
     </div>
 
+    {% if queries %}
+      <div class="alert alert-info">
+        {% trans 'Base Query' %}
+        <p>{{ queries }}</p>
```

Comment:
Might be more readable if you put each query in its own paragraph.

Reply:
Honestly i was not sure if this would ever be more than one query, so could not pick the first one instead as well.
Anyway, https://github.com/dimagi/commcare-hq/pull/26397